README.md

```diff
@@ -1,69 +1,69 @@
 # SneakerNet: resilient asynchronous networks for adverse environments.
 
-Abstract: In mountainous or sparsely populated places with little infrastructure communication and power can be a challenge. The use of portable battery packs as basic power infrastructure opens up the potential of adding communication to these batteries which is always on. Making each battery as small store and forward server using spectrum diverse low power wireless communication technologies to synchronize data between nodes as they physically move around during normal usage cycle and over low bandwidth LoRa links. Allowing for basic text based systems akin to the dial-up days of BBSs in the early 90s. 
+Abstract: For cities located in mountainous and sparsely populated places, the lack of infrastructure makes accessing telecommunications and power a challenge. The use of portable battery packs as basic power infrastructure opens up the potential of adding always-on communication to the portable battery packs. Attaching always-on telecom devices on each battery pack enables each battery to act as a small store and forward server, using spectrum diverse low power wireless communication technologies to synchronize data between battery pack nodes. Over time, portable battery packs are physically moved by holders during normal usage cycle and over low bandwidth LoRa links, allowing for basic text-based systems akin to the dial-up days of BBSs in the early 90s. 
 
-## Overview of technologies 
-The cost of lithium portable electrical storage continues to drop and become more and more robust with the roll out of LiFePO4 in many devices. 
-With these developments there are companies such as LiquidStar are providing meaningful power to communities beyond the grid with a current pilot in Djibouti deploying 75 of these batteries.
-55 of these batteries have built in tracking,logging, and communication as they are intended to be rented out and gather usage data. 
-Tech includes LoRa radio, ESP32, Swarm satellite comms, 4gb SD card, GPS, and an expansion port that can be used for 4g radio or similar device. 
+## Overview of technologies used:
+
+The cost of lithium portable electrical storage continues to drop and become more and more robust with the rollout of LiFePO4 in many devices. 
+With these developments, there are companies like LiquidStar that provide meaningful power infrastructure to communities outside of the grid, with a current pilot in Djibouti deploying 75 batteries.
+55 of these batteries have built in tracking, logging, and telecommunication capabilities, and those capabilities are used responsive to rental as they are intended to be rented out and gather usage data. 
```

Comment:
These will not have 4g modems so wireless communications might be more apt. 

"used responsive to rental..." not sure what you are trying to say here.